mycode/iteration2.sql

```diff
@@ -0,0 +1,262 @@
+-- MySQL Workbench Forward Engineering
```

Comment:
El resultado del ERD en workbench es este código SQL. Su sentido es que puedas crear la base de datos física a partir de un diagrama de relaciones (forward engineering) que previamente has construido. Si ejecutas tu código en workbench verás que te da error en la tabla de usuario adicional, como es la primera no crea ninguna. Te adjunto pantallazo. 
Casi todos los interfaces de gestión de bases de datos te permiten tanto el forward engineer como el reverse engineer que es la situación inversa, partiendo de una base de datos física recuperar su diagrama de relaciones. 
Aunque no forma parte del ejercicio, la comprobación de este tipo de cosas es interesante y si tienes un hueco en algún momento, sin restricciones de la relación one-to-one compruébalo por tu cuenta. 
<img width="1440" alt="screen shot 2018-10-22 at 08 39 49" src="https://user-images.githubusercontent.com/22130700/47283924-9511d100-d5e5-11e8-873f-39d4ea052072.png">

No puedo comentar en la imagen  así que te doy por aquí el feedback. La base de datos que has diseñado tiene bastantes tablas y bastantes relaciones, tiene buena pinta en general, pero las tablas de usuarios, usuarios_adic y la de facturación pueden darte problemas si la fueras a crear. El error de ejecución que te da creo que está localizado en esta relación forzada de uno a uno.